### ISSUE TITLE
Question about training data

### ISSUE THREAD

**Issue by chynphh** [lifu-tu/ENGINE#2]
Did you use raw data or distilled data for training the CMLM model(for initializing the ENGINE model)? And which type of data do you use to train ENGINE?

In your  `ENGINE_WMT16ROEN.zip` file, I found that the pre-trained CMLM model got 28.20 BLEU scores on WMT16 RO-EN test dataset, which means the CMLM the trained using the  distilled dataset. But you preprocess the raw data in your `preprocess_wmtRO2EN.sh` file.

So, you trained the CMLM model using the distilled data, and then trained the ENGINE model using the raw data. Am I understanding it correctly?

**Comment by lifu-tu** [comment]
Either raw data or distilled data can be used for training the CMLM models or ENGINE models. Previous results in the paper [Mask-Predict](https://arxiv.org/abs/1904.09324) have shown that "training with model distillation substantially outperforms training on raw data".  The CMLM models used in the paper are trained with distilled output. The final ENGINE models in the paper are also trained with distilled data ( actually we did not use the distilled data, just the target sequence length ).

"preprocess_wmtRO2EN.sh" file shows an example for processing the raw data. It was used to train the autoregressive transformer model. You could process the distilled data with a minor change (just using the distilled data folder instead of the raw text fold ).

**Comment by chynphh** [comment]
I understand. Thank you very much for your reply. 
I think this is an amazing job~ : )
